plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoProjectionPushdownPlans.java

```diff
@@ -58,6 +59,7 @@ public class TestMongoProjectionPushdownPlans
     private static final String CATALOG = "mongodb";
     private static final String SCHEMA = "test";
 
+    private Closer closer;
     private MongoServer server;
     private MongoClient client;
```

Comment:
Do we need this global variable - the closing on these resources is handled by the `Closer` so I guess we could remove them.